data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py

```diff
@@ -136,13 +135,20 @@
     o.value_source_concept_id,
     o.value_source_value, 
     o.questionnaire_response_id
-FROM `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` o
+FROM `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}_deleted` o
```

Comment:
So this uses the deleted obs ids and creates a new one using the concept_ids? This isn't your logic but it feels odd. I guess it's ok as long as there are no id collisions.